Exercise_2.java

```diff
@@ -1,3 +1,5 @@
+//Time Complexity =O(n^2)

```

Comment:
Please mention avg time complexity also

Reply:
Can you tell me time and space complexity for precourse 2 for all programs

Reply:
ang TC is O(nlog n)

Reply:
Only commented ones are wrong rest all of them are right